src/Handler/CommandNameExtractor/CommandHandlerNameExtractor.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace League\Tactician\Handler\CommandNameExtractor;
+
+use League\Tactician\Exception\CanNotDetermineCommandNameException;
+
+/**
+ * Extract the name from a command so that the name can be determined
+ * by the context better than simply the class name
+ */
+interface CommandHandlerNameExtractor
+{
+    /**
+     * Extract the handler name from a command FQCN.
+     *
+     * @param string $command
+     *
```

Comment:
@mvriel 

Should we add

```php
/**
 * @throws \InvalidArgumentException
*/
```

here, since the implementation provided in this PR throws one?